python/src/aiconfig/editor/client/src/shared/types.ts

```diff
@@ -61,3 +61,6 @@ export function aiConfigToClientConfig(aiconfig: AIConfig): ClientAIConfig {
     },
   };
 }
+
+export type LogEvent = "ADD_PROMPT" | "SAVE_BUTTON_CLICKED";
+export type LogEventData = JSONObject;
```

Comment:
not a blocker, but let's add an issue to schematize the logging payload

Reply:
added a todo

Reply:
are we tracking editor issues with github issues?

Reply:
We should, TODOs were mainly from when it was pre-release

Reply:
sounds good, created #915